examples/pxScene2d/src/pxContext.h

```diff
@@ -53,7 +53,46 @@ class shaderProgram; //fwd
   #define PXSCENE_DEFAULT_TEXTURE_MEMORY_LIMIT_THRESHOLD_PADDING_IN_BYTES (5 * 1024 * 1024)
 #endif
 
-//enum pxStretch { PX_NONE = 0, PX_STRETCH = 1, PX_REPEAT = 2 };
+
+typedef struct shadowFx_
+{
+  bool      shadow;
+
+  float     shadowColor[4];
+  float     shadowOffsetX;
+  float     shadowOffsetY;
+  float     shadowBlur;
+
+  float     x;         // used internally
+  float     y;         // used internally
+  float     radius;    // used internally
+  
+  float     width;           // used internally
+  float     height;          // used internally
+}
+shadowFx_t;
+
+typedef struct highlightFx_
+{
+  bool      highlight;
+
+  float     highlightColor[4];
+  float     highlightOffset;
+  float     highlightPaddingLeft;
+  float     highlightPaddingRight;
+  
+  float     highlightHeight; // used internally
+  float     width;           // used internally
+  float     height;          // used internally
+}
+highlightFx_t;
+
+typedef struct textFx_
+{
+  highlightFx_t  highlight;
+  shadowFx_t     shadow;
+}
+textFx_t;
```

Comment:
Please move these structs to pxContextGL.cpp to make them internal concepts (please see comments below).

Reply:
Moved to *pxFont.h*